clihandler/initcliutils.go

```diff
@@ -58,11 +58,12 @@ func getHostSensorHandler(scanInfo *cautils.ScanInfo, k8s *k8sinterface.Kubernet
 	// we need to determined which controls needs host sensor
 	if scanInfo.HostSensor.Get() == nil && hasHostSensorControls {
 		scanInfo.HostSensor.SetBool(askUserForHostSensor())
+		cautils.WarningDisplay(os.Stdout, "Warning: K8S nodes scanning is disabled. This is required to collect valuable data for certain controls. You can enable it using  the --enable-host-scan flag\n")
```

Comment:
1. Use `os.Stderr` instead of `os.Stdout`
2. Write `Warning: K8S nodes` -> `Warning: Kubernetes cluster nodes`